docs/concepts.rst

```diff
@@ -1,128 +1,130 @@
 
 Project Structure
 -----------------
 
-A CumulusCI project represents an application or a major component of an application on the Salesforce platform. Each project corresponds with a single managed package, or with an org-based implementation. In this documentation, we'll often speak about a package; if you're building a project that isn't packaged, please take this to refer to an unmanaged package within which your implementation is deployed.
+Project Directory
+^^^^^^^^^^^^^^^^^
 
-Together with the core package, a CumulusCI project encompasses a variety of unpackaged application components, automation, tests, and other support material. We'll quickly survey the primary components of a CumulusCI project here. Each component is covered in more depth later in this guide.
+The root directory of your CumulusCI project. Because each project is linked to a single GitHub repository, CumulusCI knows which project you are working on by the current working directory of your shell. 
 
-Automation Configuration (``cumulusci.yml``)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. tip:: Avoid headaches by making sure you're in the correct repository for your project before running project-specific commands; otherwise, your project will produce an error. (This tends to be one of the biggest troubleshooting issues that developers run into with CumulusCI. Check for this first and you might save yourself an extra trip to this guide.).
 
-``cumulusci.yml`` is the definition of the project's automation. It includes information about the project itself as well as customizations that define how scratch orgs are built and provide other important functionality to be used as part of the project's lifecycle.
+In order to be used as a CumulusCI project, a directory must both be a Git repository and contain a ``cumulusci.yml`` configuration file. We'll show you how to get set up with a new or existing CumulusCI project in the [TODO link Get Started section].
 
-When you work with CumulusCI in a project, the contents of your ``cumulusci.yml`` are merged with the ``cumulusci.yml`` that comes with CumulusCI and defines out-of-the-box functionality. From time to time we'll refer to that as the standard library. Learn more about customizing CumulusCI automation in TODO: link Customization section.
+cumulusci.yml
+^^^^^^^^^^^^^
 
-Project Source Code
-^^^^^^^^^^^^^^^^^^^
+The ``cumulusci.yml`` file defines a project's automation in CumulusCI. It contains all the customizations and configurations that pertain to your project's lifecycle. It can encompass everything from customizing the shapes of scratch orgs to configuring tasks and flows.
```

Comment:
```suggestion
The ``cumulusci.yml`` file defines a project's automation. It contains all the customizations and configurations that pertain to your project's lifecycle. It can encompass everything from customizing the shapes of scratch orgs to configuring tasks and flows.
```